trompace/client/client_itemlist.py

```diff
@@ -0,0 +1,128 @@
+
+from trompace.config import config
+from trompace.connection import submit_query
+from trompace.mutations.itemlist import *
+from trompace.constants import ItemListOrderType
+
+
+
+def create_itemlist_node(contributor = contributor, name = name, 
```

Comment:
This doesn't look right - you should just specify the name of the parameter and its type, e.g.:
`def create_itemlist_node(contributor: str, name: str, description: str = None, ordered: bool = False)`